jscomp/others/js_option.ml

```diff
@@ -59,5 +59,27 @@ let andThen f x =
   | None -> None 
   | Some x -> f x [@bs]
 
+let map f x =
+  match x with
+  | None -> None
+  | Some x -> Some (f x [@bs])
 
+let orValue a x =
+  match x with
+  | None -> a
+  | Some x -> x
```

Comment:
We should probably copy all of these:  http://ocaml-lib.sourceforge.net/doc/Option.html